packages/account/src/Sections/Security/LoginHistory/login-history.jsx

```diff
@@ -0,0 +1,177 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+import classNames from 'classnames';
+import { DesktopWrapper, MobileWrapper, Loading, DataTable } from '@deriv/components';
+import { convertDateFormat } from '@deriv/shared/utils/date';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import { WS } from 'Services/ws-methods';
+import LoadErrorMessage from 'Components/load-error-message';
+
+const Td = ({ title, text, className }) => (
+    <td>
+        <h3 className='cell-title'>{title}</h3>
+        <span className={classNames('cell-value', className)}>{text}</span>
+    </td>
+);
+
+const LoginHistoryListView = ({ fields, login_history, is_dark_mode }) => (
+    <table
+        className={classNames('login-history-table', {
+            'theme-dark': is_dark_mode,
```

Comment:
Why do we need this `className` modifier? Shouldn't using `CSS` `variable` colors be enough to handle theme switching?